src/Avalonia.Base/Utilities/MathUtilities.cs

```diff
@@ -224,6 +224,34 @@ public static double Clamp(double val, double min, double max)
             }
         }
 
+        /// <summary>
+        /// Clamps a value between a minimum and maximum value.
+        /// </summary>
+        /// <param name="val">The value.</param>
+        /// <param name="min">The minimum value.</param>
+        /// <param name="max">The maximum value.</param>
+        /// <returns>The clamped value.</returns>
+        public static decimal Clamp(decimal val, decimal min, decimal max)
```

Comment:
What's wrong with `System.Math.Clamp(Decimal, Decimal, Decimal)` ? https://docs.microsoft.com/en-us/dotnet/api/system.math.clamp?view=net-5.0#System_Math_Clamp_System_Decimal_System_Decimal_System_Decimal_

Reply:
It's not available in netstandard 2.0